### ISSUE TITLE
Bullets change direction

### ISSUE THREAD

**Issue by Graphcalibur** [Invincibot/covid-tower-defense#255]
The tracking bullets can sometimes change directions mid-flight, which looks pretty strange for cells like T-Cells and Goblet Cells. Is this intentional?

**Comment by vincetiu8** [comment]
Yes, but I am open to changing it. This is because all tracking towers, from T-Cells to M-Cells, use the same algorithm to spawn tracking bullets. The bullets target the closest enemy, which sometimes (if they miss or the original was killed) cause them to fly backwards.

The best fix for this would be to decrease the rotation speed of the bullets. Currently, they obtain this from the parent tower's rotation speed, and as t-cell and goblet-cells have an infinite rotation speed, the bullets also have that. Perhaps separating the rotation speed would be the best option.
